sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -1,12 +1,17 @@
 # Release History
 
-## 12.2.0-beta.2 (Unreleased)
+## 12.2.0 (2020-01-08)
+This package's
+[documentation](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.2.0/sdk/storage/azure-storage-blob/README.md)
+and
+[samples](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.2.0/sdk/storage/azure-storage-blob/src/samples/java/com/azure/storage/blob)
+
+- Release of 12.2.0-beta.1
```

Comment:
Were the items below part of 12.2.0-beta.1 or part of the release of 12.2.0?

Reply:
The items below were added after 12.2.0-beta.1

Reply:
Do you think it would read better if I added it at the end?

Reply:
I think you should just remove "- Release of 12.2.0-beta.1" as I'm not sure it is needed there given that the things under it were added after it. Also the version numbers are already in the headers so shouldn't be needed here.